src/main/resources/service_config.json

```diff
@@ -64,6 +64,14 @@
         "UPDATE",
         "DELETE"
       ]
+    },
+    {
+      "id": "em_npa_app",
+      "caseTypeId": ["*"],
+      "jurisdictionId": "*",
```

Comment:
does this service deals with multiple jurisdictions?

Reply:
Yes.